backend/graminsta/post/models.py

```diff
@@ -17,10 +17,17 @@ class Post(models.Model):
     lat = models.FloatField(blank=True, null=True)
     img = models.ImageField(settings.MEDIA_ROOT)
     marked_user = models.ManyToManyField(auth_models.User,
-                                         related_name="marking_users")
-    mention_user = models.ManyToManyField(
-        auth_models.User, related_name="mentioned_by_post")
+                                         related_name="marking_users",
+                                         blank=True)
+    mention_user = models.ManyToManyField(auth_models.User,
+                                          related_name="mentioned_by_post",
+                                          blank=True)
     created_at = models.DateTimeField(auto_now_add=True)
+    kudos = models.IntegerField(default=0)
+
+    def __str__(self):
+        return '%s: %s' % (self.publisher.username,
+                           self.description)
 
```

Comment:
I think you can start with:

1. My posts
2. Posts from my following people

So you don't need to make these model changes.

Reply:
Actually I only add `blank=True` feature to make `mention_user` and `marked_user` field not required, these two fields did exist before in dev branch, do I need to remove them?

Reply:
[This PR](https://github.com/SPS-2020-Team-15/Graminsta/pull/11) adds `marked_user` and `mention_user` field, I think these two fields should not be required when I created a post object to test, so I add the `blank=True` feature.

Reply:
Got it, but in order to make this field optional, you need to add `null=True` too. `blank=True` only bypass Django field validation, whereas `null=True` allows the field to be null in database level